spring-integration-file/src/main/java/org/springframework/integration/file/filters/LastModifiedFileListFilter.java

```diff
@@ -124,4 +124,21 @@ else if (this.discardCallback != null) {
 		return list;
 	}
 
+	@Override
+	public boolean accept(File file) {
+		if (file.lastModified() / ONE_SECOND + this.age <= System.currentTimeMillis()) {
```

Comment:
See `filterFiles()` logic: you are missing to divide `System.currentTimeMillis()` to the `ONE_SECOND`